main.lua

```diff
@@ -41,7 +41,14 @@ expose("ptusa", function()
 
                 obj:set_mode( i, 1 )
                 assert.are.equal( ptusa.eval(), 0 )
-
+                local m_mngr = obj:get_modes_manager()
+                local oper   = m_mngr[ i ]
```

Comment:
Убрал лишние пробелы:
```suggestion
                local oper = m_mngr[ i ]
```